core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -403,8 +390,11 @@ public static void parseDocuments(XContentParser parser, List<Item> items, @Null
                     } else if ("_parent".equals(currentFieldName) || "parent".equals(currentFieldName)) {
                         parent = parser.text();
                     } else if ("fields".equals(currentFieldName)) {
-                        fields = new ArrayList<>();
-                        fields.add(parser.text());
+                        throw new ParsingException(parser.getTokenLocation(),
+                            "Deprecated field [fields] used, expected [stored_fields] instead");
```

Comment:
I think it'd be more accurate to call it "unsupported" rather the "deprecated" because it doesn't work any more.